src/Stripe.net/Services/Sources/StripeSourceCreateOptions.cs

```diff
@@ -44,6 +44,9 @@ public class StripeSourceCreateOptions : StripeBaseOptions, ISupportMetadata
         [JsonProperty("sepa_debit[iban]")]
         public string SepaDebitIban { get; set; }
 
+        [JsonProperty("sepa_debit[ideal]")]
+        public string SepaDebitIdealSource { get; set; }
```

Comment:
I think this should be named `SepaDebitIdealSourceId` in order to be consistent with other parameters that expect an ID.